lib/util/auth.js

```diff
@@ -77,13 +70,27 @@ export function getAuth0Callbacks (reduxStore) {
      * Where to send the user after they have signed in.
      */
     onRedirectCallback: appState => {
-      if (appState && appState.returnTo) {
+      if (appState && appState.urlHash) {
         /**
-         * Immediately after successful Auth0 login, the user will be redirected to the MOD UI URL with no search.
-         * We want to redirect to the URL just before the Auth0 operation (stored in appState.returnTo),
+         * Immediately after successful login, Auth0 will redirect to a URL (in whitelist) with no search parameters.
+         * If the user has set up their MOD UI account, we want to redirect to
+         * the state just before the Auth0 operation (stored in appState.urlHash),
          * and prevent the URL with no search from appearing in the user's browsing history.
          */
```

Comment:
For multi-line comments that are not jsdoc, I prefer not to use this style. Instead, please use the double slash `//`